examples/opengl_underlay/scene.60

```diff
@@ -0,0 +1,34 @@
+// Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+// SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
+
+import { ScrollView, Button, CheckBox, SpinBox, Slider, GroupBox, LineEdit, StandardListView,
+    ComboBox, HorizontalBox, VerticalBox, GridBox, TabWidget, TextEdit, AboutSixtyFPS } from "sixtyfps_widgets.60";
+
+App := Window {
+    preferred-width: 500px;
+    preferred-height: 600px;
+    title: "SixtyFPS Gallery";
+    icon: @image-url("../../vscode_extension/extension-logo.png");
+
+    VerticalBox {
+        Rectangle {
+            background: #ffffff92;
+            HorizontalBox {
+                Text {
+                    text: "This text is rendered using SixtyFPS. The animation below is rendered using OpenGL code.";
+                    wrap: word-wrap;
+                }
+
+                VerticalLayout {
+                    alignment: start;
+                    disable-toggle := CheckBox {
```

Comment:
```suggestion
                    CheckBox {
```

I was wondering what the checkbox was setting.  Maybe you could make it so that it change something in the rendering

Reply:
Yes, I was thinking as well - we could add functionality. How about toggling whether to rotate the entire "thing" or not?

Reply:
This is implemented now.